### PR TITLE
New package: postgis-2.4.4

### PR DESCRIPTION
**Pull request by ArniDagur**
I built and installed the package on my own machine and it worked fine. This is my first package though, so take a look first before merging.

### CODE REVIEW

File: srcpkgs/postgis/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'postgis'
+pkgname=postgis
+version=2.4.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="perl"
+makedepends="geos proj libgdal libxml2 json-c postgresql postgresql-libs-devel libxml2-devel geos-devel proj-devel libgdal-devel libgdal-tools"
+depends=""
```
Comment (maxice8) [inline srcpkgs/postgis/template]: remove empty variable.

File: srcpkgs/postgis/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'postgis'
+pkgname=postgis
+version=2.4.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="perl"
+makedepends="geos proj libgdal libxml2 json-c postgresql postgresql-libs-devel libxml2-devel geos-devel proj-devel libgdal-devel libgdal-tools"
```
Comment (maxice8) [inline srcpkgs/postgis/template]: -devel packages automatically include the main package.

File: srcpkgs/postgis/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+depends=""
+short_desc="A spatial database extender for PostgreSQL object-relational database"
```
Comment (maxice8) [inline srcpkgs/postgis/template]: I think only `PostgreSQL` fits instead of the whole name

**Comment by maxice8** [comment]
Everything should be in 1 commit.

File: srcpkgs/postgis/template [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'postgis'
+pkgname=postgis
+version=2.4.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="perl"
+makedepends="geos proj libgdal libxml2 json-c postgresql-libs-devel libxml2-devel geos-devel proj-devel libgdal-devel libgdal-tools"
```
Comment (maxice8) [inline srcpkgs/postgis/template]: don't include the package without -devel if you have it as -devel

**Comment by ArniDagur** [comment]
It only seems to build correctly on x86_64 and i686. https://travis-ci.org/voidlinux/void-packages/builds/361444499

**Comment by maxice8** [comment]
Read the build log, it can't find the binary `pg_config` , which is provided by `postgresql-libs-devel`.

Since it is a binary to be ran on the system that builds it needs to be added to `hostmakedepends`

Read https://github.com/voidlinux/void-packages/blob/master/Manual.md and the build logs.

**Pull request closed without merging**